src/test/integration/org/apache/cassandra/sidecar/routes/SnapshotsHandlerIntegrationTest.java

```diff
@@ -216,7 +220,7 @@ private String createTestTableAndPopulate(CassandraTestContext cassandraTestCont
     private void assertNotFoundOnDeleteSnapshot(VertxTestContext context, String testRoute) throws InterruptedException
     {
         WebClient client = WebClient.create(vertx);
-        client.delete(config.getPort(), "localhost", testRoute)
+        client.delete(config.getPort(), "127.0.0.1", testRoute)
```

Comment:
This changes `localhost` to `127.0.0.1`. The other change is the opposite direction. Why are they made so?

Reply:
Depends on the context - in a multi-node in-jvm test module you need IP addresses... in unit tests where we're mocking things it was set to `localhost` and changing it broke stuff, so I reverted some of them back. The two sets could likely be consolidated at some point but right now that's the split required to get tests to pass in all cases.

Reply:
I see. `CassandraTestContext` builds the `InstancesConfig` from in-jvm instances that use IP addresses.